cloud: update docs for serverless billing (#13335)
pingcap/docs: tidb-cloud/serverless-tier-faqs.md

### Is Serverless Tier free during beta?

> Until May 30th, 2023, Serverless Tier clusters are still free, with a 0% discount on usage. Starting from June 1st, 2023, usage beyond the free quota will be charged. Once the free quota limit of a cluster is reached, access to the cluster will be restricted until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters).

Author: ```suggestion
Until May 30th, 2023, Serverless Tier clusters are still free, with a 0% discount on usage. Starting from June 1st, 2023, usage beyond the free quota will be charged. 
Once the free quota of a cluster is reached, the read and write operation on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) or the current usage is reset after the next month.
```

> Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount on usage. After that, usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) or the current usage is reset after the next month.

Author: ```suggestion
Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount on usage. After that, usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md) or the current usage is reset after the next month.
```

> Until May 30th, 2023, Serverless Tier clusters are still free, with a 0% discount on usage. Starting from June 1st, 2023, usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operation on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) or the current usage is reset after the next month.

Author: ```suggestion
Until May 30th, 2023, Serverless Tier clusters are still free, with a 0% discount on usage. Starting from June 1st, 2023, usage beyond the free quota will be charged. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you [get more usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) or the current usage is reset after the next month.
```

For more information, see [Serverless Tier usage quota](/tidb-cloud/select-cluster-tier.md#usage-quota).

### What does it mean for beta release?

Serverless Tier is in beta while we continuously add new features and improve existing features before it becomes generally available. We do not provide SLA for beta products. Therefore, Serverless Tier should **NOT** be used in production currently.

### What are the limitations of a Serverless Tier cluster in beta?

For each TiDB Cloud account, you can create a maximum of five complimentary Serverless Tier clusters during the beta phase.

Some of TiDB Cloud features are partially supported or not supported on Serverless Tier. See [Serverless Tier Limitations](/tidb-cloud/serverless-tier-limitations.md) for details.

### What can Serverless Tier be used for?

You can use your Serverless Tier cluster for non-production workloads such as prototype applications, development environments, hackathons, and academic courses, or to provide temporary data service for your datasets.

> ### I created a Developer Tier cluster before Serverless Tier was available. Can I still use my cluster?

Reviewer: Should Developer Tier be Dedicated Tier?

Author: 这里的 Developer Tier 是对的，是给之前还没从  Developer Tier 迁移到 Serverless Tier 集群的用户看的
